Tests/test_AlignIO_FastaIO_unittest.py

```diff
@@ -228,8 +228,11 @@ def test_example(self):
         self.assertEqual(len(alignments), 4)
         self.assertEqual(len(alignments[0]), 2)
         for a in alignments:
-            self.assertEqual(2, len(a))
-            self.assertEqual(108, a.get_alignment_length())
+            rows = (2, 3, 4, 5)
+            cols = (108, 64, 123, 456)
+            for a, rows, cols in zip(alignments, rows, cols):
```

Comment:
You've accidentally got a double nested for loop here, there should be just one loop with ``a``.